static/js/apps/eval_rig/app.tsx

```diff
@@ -136,8 +139,16 @@ export function App(props: AppPropType): JSX.Element {
           />
         )}
         {user && <p>Signed in as {user.email}</p>}
-        {allQuery["1"] && allCall["1"] && (
-          <QuerySection doc={doc} query={allQuery["1"]} calls={allCall["1"]} />
+        {user && allQuery["1"] && allCall["1"] && (
```

Comment:
nit: can "1" be a constant

Reply:
This will need to be changed with logic to choose the landing query index in follow up PR.